neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -122,25 +184,101 @@ def create_experiment(self,
             else:
                 raise
 
+        experiment = self._convert_to_experiment(api_experiment, project)
+        # Initialize new experiment
+        self._execute_alpha_operation(
+            experiment=experiment,
+            operations=self._get_init_experiment_operations(entrypoint, params, tags),
+        )
+        return experiment
+
+    @with_api_exceptions_handler
+    def send_channels_values(self, experiment, channels_with_values):
+        send_operations = []
+        for channel_with_values in channels_with_values:
+            # TODO: handle other data types
+            # points = [Point(
+            #     timestampMillis=int(value.ts * 1000.0),
+            #     x=value.x,
+            #     y=Y(numericValue=value.y.get('numeric_value'),
+            #         textValue=value.y.get('text_value'),
+            #         inputImageValue=value.y.get('image_value'))
+            # ) for value in channel_with_values.channel_values]
+            ch_values = [
+                alpha_operation.LogStrings.ValueType(
+                    value=value.y.get('text_value'),
+                    step=None,
+                    ts=int(value.ts),
```

Comment:
Make sure we should not multiply `value.ts` by 1000. We probably should or we should not convert it to int. Api expects millis. However I do not remember what is expected in `LogStrings.ValueType`.

Reply:
Good catch! Sorry for that...